src/Microsoft.Restier.EntityFramework/Model/ModelMapper.cs

```diff
@@ -60,10 +61,10 @@ public ModelMapper(Type dbContextType)
             if (property != null)
             {
                 var type = property.PropertyType;
-                if (type.IsGenericType &&
-                    type.GetGenericTypeDefinition() == typeof(DbSet<>))
+                var parent = GetGenericParent(type);
```

Comment:
You can link the `TypeExtensions` class to  `Microsoft.Restier.EntityFramework.csproj`.  An example: https://github.com/OData/RESTier/blob/master/src/Microsoft.Restier.EntityFramework/Microsoft.Restier.EntityFramework.csproj#L40-L45